apps/src/templates/codeWorkspace.html.ejs

```diff
@@ -0,0 +1,46 @@
+<%
+var msg = require('../locale');
+%>
+
+<div id="codeWorkspaceWrapper">
+  <div id="headers" dir="<%= data.localeDirection %>">
+    <div id="codeModeHeaders">
+      <div id="toolbox-header" class="workspace-header">
+        <i id="hide-toolbox-icon" style="display:none;" class="fa fa-chevron-circle-right"></i>
+        <span><%= data.editCode ? msg.toolboxHeaderDroplet() : msg.toolboxHeader() %></span>
+      </div>
+      <div id="show-toolbox-header" class="workspace-header" style="display:none;">
+        <i id="show-toolbox-icon" class="fa fa-chevron-circle-right"></i>
+        <span><%= msg.showToolbox() %></span>
+      </div>
+      <div id="show-code-header" class="workspace-header workspace-header-button"><span><img src="<%= assetUrl('media/applab/blocks_glyph.gif') %>" class="blocks-glyph" /><i class="fa fa-code"></i><%= msg.showCodeHeader() %></span></div>
+      <% if (!data.readonlyWorkspace) { %>
```

Comment:
Ah, no, I missed that.  Autoreformat on paste in RubyMine rearing its (normally useful) head.  Any suggestions on fixing it's behavior in these sort of cases?